README.md

```diff
@@ -99,31 +65,30 @@ To use the Material theme, import the correspondent file from the `theme/materia
 
 - The component with the Lumo theme:
 
-  `theme/lumo/vaadin-select.html`
+  `theme/lumo/vaadin-select.js`
 
 - The component with the Material theme:
 
-- `theme/material/vaadin-select.html`
+- `theme/material/vaadin-select.js`
 
-- Alias for `theme/lumo/vaadin-select.html`:
+- Alias for `theme/lumo/vaadin-select.js`:
 
-- `vaadin-select.html`
+- `vaadin-select.js`
 
 
 ## Running demos and tests in a browser
 
 1. Fork the `vaadin-select` repository and clone it locally.
 
-1. Make sure you have [npm](https://www.npmjs.com/) and [Bower](https://bower.io) installed.
+1. Make sure you have [npm](https://www.npmjs.com/) installed.
 
-1. When in the `vaadin-select` directory, run `npm install` and then `bower install` to install dependencies.
+1. When in the `vaadin-select` directory, run `npm install` to install dependencies.
 
 1. Run `npm start`, browser will automatically open the component API documentation.
 
-1. You can also open demo or in-browser tests by adding **demo** or **test** to the URL, for example:
+1. You can also open demo or in-browser tests by adding **test** to the URL:
 
-  - http://127.0.0.1:3000/components/vaadin-select/demo
-  - http://127.0.0.1:3000/components/vaadin-select/test
+  - http://127.0.0.1:3000/components/vaadin-select/test/
```

Comment:
This doesn't seem to work.

Reply:
Thanks, fixed.

Reply:
Now the instructions were simply removed. Is there generally some way to run the in-browser tests with the new setup?

Reply:
Added the note on `npm run debug` script and using manual mode of Web Test Runner.